addon/kubeflow-operator/manifest.yaml

```diff
@@ -242,6 +246,20 @@ spec:
         name: manifests
         path: cert-manager/cert-manager
     name: cert-manager
+  - kustomizeConfig:
+      parameters:
+      - name: email
+        value: {{ .TemplateData.ClusterData.OwnerEmail }}
```

Comment:
Have you actually verified that this works?
I think it should be `{{ .Cluster.OwnerEmail }}` 
(based on my experience here: https://github.com/kubermatic/kubecarrier/blob/master/config/addon/kubecarrier/service-cluster.yaml#L51)

Reply:
oh, great, I tried to find any examples and haven't